augur/clades.py

```diff
@@ -205,8 +228,10 @@ def run(args):
 
     clade_designations = read_in_clade_definitions(args.clades)
 
-    clade_membership = assign_clades(clade_designations, all_muts, tree, ref)
+    (basal_clade_nodes, clade_membership) = assign_clades(clade_designations, all_muts, tree, ref)
+
+    node_data = create_node_data_structure(basal_clade_nodes, clade_membership, args)
```

Comment:
The logic of `create_node_data_structure` would be easier to follow if that code lived here in the `run` function. We don't benefit much from abstracting the process of building up a node data dict, but we do lose a bit in readability by moving the code to a separate function.

Most other augur commands already build up node data dicts in a similar way in `run`, so that pattern seems fine to follow here, too.

Reply:
Done 👍